Gemfile.lock

```diff
@@ -372,4 +372,4 @@ DEPENDENCIES
   will_paginate-bootstrap (>= 1.0.1)
 
 BUNDLED WITH
-   1.14.6
```

Comment:
Hmm, looks like it's still registering some kind of change. Maybe the best way would be to download the raw version from the master publiclab branch and just copy it over your local version, and re-add it in a new commit. Sorry, this is a pain!